tests/dataset/test_sqlalchemydataset.py

```diff
@@ -347,3 +347,21 @@ def test_expect_compound_columns_to_be_unique(sa):
     assert dataset.expect_compound_columns_to_be_unique(
         ["col1", "col2", "col4"]
     ).success
+
+
+def test_expect_compound_columns_to_be_unique_with_no_rows(sa):
```

Comment:
Should I add tests to [expect_compound_columns_to_be_unique.json](https://github.com/great-expectations/great_expectations/blob/develop/tests/test_definitions/multicolumn_map_expectations/expect_compound_columns_to_be_unique.json) in addition to or instead of this test?

Reply:
Adding the JSON test is a good idea, but I am ready to merge this PR as is.